third-party/realdds/include/realdds/topics/flexible-msg.h

```diff
@@ -75,10 +75,10 @@ class flexible_msg
                            eprosima::fastdds::dds::SampleInfo * optional_info = nullptr );
 
     // WARNING: this moves the message content!
-    raw::flexible to_raw();
+    raw::flexible to_raw() &&;
```

Comment:
Why do we need to add this and cast to rvalue reference all users? Isn't the content being moved as it is?

Reply:
Yes, I wanted to force the user to use `std::move` to make it absolutely clear we're moving the object.

Reply:
OK